packages/components/src/Tooltip/_docs/Tooltip.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import { Decorator, Meta, StoryFn, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic/isChromatic"
+import { Tag } from "../../Tag"
+import { Button, InformationIcon, Text } from "../../index"
+import { Tooltip } from "../index"
+
+const openTooltipInChromatic: Decorator = (Story, { args }) => {
+  if (isChromatic()) args.isInitiallyVisible = true
+  return <Story />
+}
```

Comment:
This isn't necessary here since it's only for stories sent to Chromatic, which have now moved to stickersheet.